config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  root 'home#index' 
-# The priority is based upon order of creation: first created -> highest priority.
+  root 'home#index'
+  resources :cards, only: :index
+# The priority is based upon order of creation: first created -> highest priority
```

Comment:
Line is too long. [81/80]<br>Incorrect indentation detected (column 0 instead of 2).